content/docs/configuration/acme/README.md

```diff
@@ -80,6 +83,41 @@ Solvers come in the form of [`dns01`](./dns01/README.md) and
 these solver types, visit their respective documentation -
 [DNS01](./dns01/README.md), [HTTP01](./http01/README.md).
 
+### ACME Certificate Profiles
+
+> ℹ️ This feature is available in cert-manager `>= v1.18.0`.
+
+An ACME Server *may* offer a selection of different certificate profiles to ACME Clients.
+
+Use the optional `profile` field in the `Issuer` or `ClusterIssuer` to select a profile for your ACME orders.
+
+For example, Let's Encrypt offers the following [profiles](https://letsencrypt.org/docs/profiles/):
+
+- [`classic`][classic]: is the default profile selected for all orders which do not request a specific profile
+- [`tlsserver`][tlsserver]: for standard server certificates.
+- [`shortlived`][shortlived]: for short-lived six-day certificates.
+
+> ⚠️ The `shortlived` profile is currently locked behind an allow list.
+
+[classic]: https://letsencrypt.org/docs/profiles/#classic
+[tlsserver]: https://letsencrypt.org/docs/profiles/#tlsserver
+[shortlived]: https://letsencrypt.org/docs/profiles/#shortlived
```

Comment:
suggestion: I'm a little worried about documenting these here because it seems so easy for this to get out of date. It's not a blocker but I feel like we could save ourselves some work by just linking to Let's Encrypt documentation and not listing them here. That wouldn't prevent us using `tlsserver` as an example.

What do you think?

Reply:
Done.